code/modules/mob/living/carbon/human/species/station.dm

```diff
@@ -517,6 +518,63 @@
 		"is cracking their exoskeleton!",
 		"is stabbing themselves with their mandibles!",
 		"is holding their breath!")
+
+/datum/species/moth
+	name = "Lymantria"
+	name_plural = "Lymantrias"
+	icobase = 'icons/mob/human_races/r_moth.dmi'
+	deform = 'icons/mob/human_races/r_def_moth.dmi'
+	path = /mob/living/carbon/human/moth
+	default_language = "Galactic Common"
+	language = "Gypsikin"
+	unarmed_type = /datum/unarmed_attack/claws
+	tail = "mothwings"
+
+
+	clothing_flags = HAS_UNDERWEAR | HAS_UNDERSHIRT | HAS_SOCKS
+	bodyflags = FEET_CLAWS | HAS_HEAD_ACCESSORY | HAS_SKIN_COLOR | HAS_BODY_MARKINGS | HAS_TAIL | HAS_TAIL_MARKINGS
+	eyes = "moth_eyes_s"
+	dietflags = DIET_HERB
+	blood_color = "#ffff00"
+	reagent_tag = PROCESS_ORG
+	butt_sprite = "kidan" //need to replace this when we get a buttsprite for them
+	default_headacc = "Moth Antennae"
+
+	has_organ = list(
+		"heart" =    /obj/item/organ/internal/heart,
+		"lungs" =    /obj/item/organ/internal/lungs,
+		"liver" =    /obj/item/organ/internal/liver,
+		"kidneys" =  /obj/item/organ/internal/kidneys,
+		"brain" =    /obj/item/organ/internal/brain,
+		"appendix" = /obj/item/organ/internal/appendix,
+		"eyes" =     /obj/item/organ/internal/eyes,
+		)
+
+	suicide_messages = list(
+		"is jamming their claws into their eye sockets!",
+		"is twisting their own neck!",
+		"is cracking their exoskeleton!",
+		"is holding their breath!")
+
+/datum/species/moth/handle_life(var/mob/living/carbon/human/H)
+
+	var/light_amount = 0 //how much light there is in the place
+	if(isturf(H.loc)) //else, there's considered to be no light
+		var/turf/T = H.loc
+		light_amount = min(T.get_lumcount() * 10, 5)  //hardcapped so it's not abused by having a ton of flashlights
+
+
+	if(light_amount >= 5) //if there's enough light, heal
+		slowdown = 0
+		H.clear_alert("nolight")
+	else if (light_amount >= 3)
+		slowdown = (5-light_amount)
```

Comment:
I think if you do `slowdown = max(5 - light_amount, 3)` here, you can combine this `else if ... else` into a single `else` statement